lib/reader.js

```diff
@@ -51,23 +30,22 @@ module.exports = async feeds => {
         'Expected at least 1 feed with more than one entry'
     );
 
-    const rows = getRows(feeds);
+    const opts = {
+        directory: await directory({ unsafeCleanup: true }),
```

Comment:
This will always create the tmp dir even if the consumer passes it in. Should check if it's passed before executing this function